plugin-server/src/utils/db/db.ts

```diff
@@ -1134,17 +1134,18 @@ export class DB {
         return insertResult.rows[0]
     }
 
-    public async doesPersonBelongToCohort(cohortId: number, person: IngestionPersonData): Promise<boolean> {
+    public async doesPersonBelongToCohort(cohortId: number, personUuid: string, teamId: number): Promise<boolean> {
         const psqlResult = await this.postgresQuery(
             `
             SELECT count(1) AS count
             FROM posthog_cohortpeople
             JOIN posthog_cohort ON (posthog_cohort.id = posthog_cohortpeople.cohort_id)
+            JOIN (SELECT * FROM posthog_person where team_id = $3) AS posthog_person_in_team ON (posthog_cohortpeople.person_id = posthog_person_in_team.id)
```

Comment:
checked it works in metabase http://metabase-eu/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiItLSBTRUxFQ1QgY291bnQoMSkgQVMgY291bnRcbnNlbGVjdCAqXG4gICAgICAgICAgICBGUk9NIHBvc3Rob2dfY29ob3J0cGVvcGxlXG4gICAgICAgICAgICBKT0lOIHBvc3Rob2dfY29ob3J0IE9OIChwb3N0aG9nX2NvaG9ydC5pZCA9IHBvc3Rob2dfY29ob3J0cGVvcGxlLmNvaG9ydF9pZClcbiAgICAgICAgICAgIEpPSU4gKFNFTEVDVCAqIEZST00gcG9zdGhvZ19wZXJzb24gd2hlcmUgdGVhbV9pZCA9IDMwMDIpIEFTIHBvc3Rob2dfcGVyc29uX2luX3RlYW0gT04gKHBvc3Rob2dfY29ob3J0cGVvcGxlLnBlcnNvbl9pZCA9IHBvc3Rob2dfcGVyc29uX2luX3RlYW0uaWQpXG4gICAgICAgICAgICBXSEVSRSBjb2hvcnRfaWQ9MTI2NVxuICAgICAgICAgICAgLS0gICBBTkQgcG9zdGhvZ19wZXJzb25faW5fdGVhbS5pZD03MzcyNzY0MFxuICAgICAgICAgICAgICBBTkQgcG9zdGhvZ19wZXJzb25faW5fdGVhbS51dWlkPScwMTg2YWVhNS0zYzMxLTAwMDAtNGFjOS03ODY3YjVmYjhmODknXG4gICAgICAgICAgICAtLSAgIEFORCB0ZWFtX2lkPTMwMDJcbiAgICAgICAgICAgIC0tICAgQU5EIHBlcnNvbl9pZD03MzcyNzY0MFxuICAgICAgICAgICAgICBBTkQgcG9zdGhvZ19jb2hvcnRwZW9wbGUudmVyc2lvbiBJUyBOT1QgRElTVElOQ1QgRlJPTSBwb3N0aG9nX2NvaG9ydC52ZXJzaW9uXG5saW1pdCA1IiwidGVtcGxhdGUtdGFncyI6e319LCJkYXRhYmFzZSI6MzR9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319